proxy/src/main/java/com/velocitypowered/proxy/config/AnnotatedConfig.java

```diff
@@ -65,76 +62,91 @@ public static Logger getLogger() {
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target({ElementType.FIELD, ElementType.TYPE})
-    public @interface IsMap {
-    }
+    public @interface IsMap {}
 
     /**
      * Indicates that a field is a string converted to byte[]
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target({ElementType.FIELD, ElementType.TYPE})
-    public @interface StringAsBytes {
-    }
+    public @interface StringAsBytes {}
 
     /**
-     * Indicates that a field should be skiped
+     * Indicates that a field should be skipped
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target({ElementType.FIELD})
-    public @interface Ignore {
-    }
+    public @interface Ignore {}
 
+    /**
+     * Dumps configuration to list of strings
```

Comment:
This should be more descriptive, I can see this by looking at the signature of the method. A comment (or doc string) should tell you what the code/signature can't.

How does it dump it? `key=value` representation? `key: value`? I'm getting nothing from this description...

Reply:
I left that part undone, it was in my mind to make it more descriptive in such way yep. :smile: